egi/api_resources/gamelisting/checkin.py

```diff
@@ -46,11 +46,20 @@
 )
 
 post_parser.add_argument(
-    'connected_player_count', dest='connected_player_count', type=int,
+    'connected_account_count', dest='connected_player_count', type=int,
     location='form', required=False,
 )
 post_parser.add_argument(
-    'total_player_count', dest='total_player_count', type=int,
+    'total_account_count', dest='total_player_count', type=int,
+    location='form', required=False,
+)
+
+post_parser.add_argument(
+    'connected_player_count', dest='fallback_connected_player_count', type=int,
```

Comment:
We could also match here as `connected_player_count`, but with a comment on the preceding line that this is a backwards compat shim for the 0.6->0.7 transition.